examples/test/post-processing/components/bloom.js

```diff
@@ -0,0 +1,161 @@
+/* global AFRAME THREE */
+
+/*
+Kawase bloom, described here
+https://software.intel.com/en-us/blogs/2014/07/15/an-investigation-of-fast-real-time-gpu-based-image-blur-algorithms
+*/
+
+AFRAME.registerComponent('bloom', {
+
+  schema: {
+    passes: { default: '0 1 2 3 4 5 7' },
+    threxp: { type: 'vec2', default: new THREE.Vector2(16, 0) },
+    intensity: { type: 'float', default: 1.0 }
+  },
+
+  setupPostState: function () {
+    this.scenePost = new THREE.Scene();
+    this.cameraPost = new THREE.OrthographicCamera(-1, 1, 1, -1, 0, 1);
+    this.quadPost = new THREE.Mesh(new THREE.PlaneBufferGeometry(2, 2), null);
+    this.scenePost.add(this.quadPost);
+  },
+
+  init: function () {
+    this.setupPostState();
+    this.brightMaterial = new THREE.ShaderMaterial({
+      uniforms: {
+        srcTexture: { type: 't', value: null },
+        threxp: { type: 'v2', value: new THREE.Vector2(0, 0) }
+      },
+      vertexShader: this.vertexBrightText,
+      fragmentShader: this.fragmentBrightText,
+
+      depthWrite: false
+
+    });
+    this.blurMaterial = new THREE.ShaderMaterial({
+      uniforms: {
+        iteration: { type: 'f', value: 0 },
+        srcTexture: { type: 't', value: null },
+        pixelSize: { type: 'v2', value: new THREE.Vector2(0, 0) }
+      },
+      vertexShader: this.vertexBlurText,
+      fragmentShader: this.fragmentBlurText,
+
+      depthWrite: false
+
+    });
+    this.quadPost.material = this.brightMaterial;
+    var rtOptions = { minFilter: THREE.LinearFilter, magFilter: THREE.NearestFilter, format: THREE.RGBAFormat };
```

Comment:
better use whole words for variables instead of abbreviations: `renderTargetOptions`